packages/ui/src/services/component-config/component-config.ts

```diff
@@ -1,3 +1,26 @@
+import { useLocalConfig } from '../../components/va-config/VaConfig'
+import { useGlobalConfig } from '../global-config/global-config'
+import { computed, DefineComponent, ref } from 'vue'
+
 // TODO ideally we want to type component props instead  of any
```

Comment:
We don't need this comment anymore, because now withTransportConfig return component type instead of any.